wasm-geopolars/Cargo.toml

```diff
@@ -26,7 +26,8 @@ console_error_panic_hook = { version = "0.1.6", optional = true }
 # Unfortunately, `wee_alloc` requires nightly Rust when targeting wasm for now.
 wee_alloc = { version = "0.4.5", optional = true }
 
-geopolars = { path = "../geopolars", default-features = false }
+geopolars = { path = "../geopolars", default-features = false, features = ["basic-polars"] }
```

Comment:
Can we add a dependency on polars here so that the polars features needed by wasm are defined here instead of in `geopolars/Cargo.toml`?